frontend/src/metabase/visualizations/visualizations/PivotTable.jsx

```diff
@@ -165,30 +168,32 @@ export default class PivotTable extends Component {
       rowCount,
       columnCount,
     } = pivoted;
-    const cellWidth = 100;
-    const cellHeight = 25;
+    const CELL_WIDTH = 100;
+    const CELL_HEIGHT = 25;
```

Comment:
Any reasons these consts are defined again?

Reply:
Nope! I removed it.